Kitsunebi/Classes/AnimationView.swift

```diff
@@ -64,8 +64,9 @@ open class PlayerView: UIView {
     gpuLayer.drawsAsynchronously = true
     gpuLayer.contentsGravity = .resizeAspectFill
     gpuLayer.pixelFormat = .bgra8Unorm
-    gpuLayer.framebufferOnly = false
+    gpuLayer.framebufferOnly = true
     gpuLayer.presentsWithTransaction = false
+    (gpuLayer as? CAMetalLayer)?.device = MTLCreateSystemDefaultDevice()
```

Comment:
Really to need create device?
```suggestion
    (gpuLayer as? CAMetalLayer)?.device = device
```